src/main/kotlin/christmas/domain/order/Order.kt

```diff
@@ -0,0 +1,30 @@
+package christmas.domain.order
+
+import christmas.domain.menu.Category
+import christmas.domain.menu.Menu
+import christmas.domain.menu.Money
+
+data class Order(val orders: List<Menu>) {
+
+    init {
+        require(orders.size <= 20) { " Order must not greater than 20 orders" }
+        require(orders.filter { order -> order.category == Category.BEVERAGE }.size != orders.size) {
+            "you can not order beverage only"
+        }
```

Comment:
```suggestion
        if (orders.all { it.category == Category.BEVERAGE }) {
            throw IllegalArgumentException("you can not order beverage only")
        }
```

I think using `require` makes the code less readable. What do you think?